File: tests/lambda/js/list_s3_buckets.js
import {S3Client, ListBucketsCommand} from "@aws-sdk/client-s3"

const s3 = new S3Client({})

export const lambda_handler = async (event, context) => {
  try {
    const {Buckets} = await s3.send(new ListBucketsCommand({}))
    const bucketNames = Buckets.map(bucket => bucket.Name)
    const result = bucketNames.join('\n')
    return result
  } catch (err) {
    return err
  }
}

File: tests/lambda/py/list_s3_buckets.py
import boto3


def lambda_handler(event, context):
  s3 = boto3.client('s3')
  try:
    buckets_data = s3.list_buckets()
    bucket_names = [bucket['Name'] for bucket in buckets_data['Buckets']]
    result = '\n'.join(bucket_names)
    return result
  except Exception as ex:
    return ex
